### PR TITLE
Adding Whatson and Sheffield.Digital slack to announcment

### PR DESCRIPTION
**Pull request by ns-rse**
Improves structure a bit too and corrects structure of nested lists.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/meeting_setup.md [hunk 1]
```diff
@@ -18,24 +20,30 @@ Steps to take when setting up a new meeting...
...
-+ [ ] **Announce on RSE Mailing List** : Announce the meeting on the [RSE Sheffield Mailing
++ [ ] **RSE Mailing List** : Announce the meeting on the [RSE Sheffield Mailing
       List](https://groups.google.com/a/sheffield.ac.uk/g/RSE-group)) mailing list including both the Google Meet and
       Calendar links.
-+ [ ] **Announce on NHS-R Slack** : If you are a member you can announce the event on the [NHS-R
-      Slack](https://nhsrcommunity.slack.com/).
-+ [ ] **Announce on Sheffield Digital Slack** : If you are a member you can announce the event on the [Sheffield Digital
-      Slack](https://sheffielddigital.slack.com).
++ [ ] **University Website - WhatsOn** : Submit [a form]( https://www.sheffield.ac.uk/whatson/submit ) with details of
+      the event to announce on the University's Whats on public page.
```
Comment (tdjames1) [inline .github/ISSUE_TEMPLATE/meeting_setup.md]:
```suggestion
+ [ ] **University Website - What's on** : Submit [a form]( https://www.sheffield.ac.uk/whatson/submit ) with details of
      the event to announce on the University's What's on public page.
```

File: .github/ISSUE_TEMPLATE/meeting_setup.md [hunk 2]
```diff
@@ -18,24 +20,30 @@ Steps to take when setting up a new meeting...
...
       Calendar links.
-+ [ ] **Announce on RSE Mailing List** : Announce the meeting on the [RSE Sheffield Mailing
++ [ ] **RSE Mailing List** : Announce the meeting on the [RSE Sheffield Mailing
       List](https://groups.google.com/a/sheffield.ac.uk/g/RSE-group)) mailing list including both the Google Meet and
       Calendar links.
-+ [ ] **Announce on NHS-R Slack** : If you are a member you can announce the event on the [NHS-R
-      Slack](https://nhsrcommunity.slack.com/).
-+ [ ] **Announce on Sheffield Digital Slack** : If you are a member you can announce the event on the [Sheffield Digital
-      Slack](https://sheffielddigital.slack.com).
++ [ ] **University Website - WhatsOn** : Submit [a form]( https://www.sheffield.ac.uk/whatson/submit ) with details of
```
Comment (tdjames1) [inline .github/ISSUE_TEMPLATE/meeting_setup.md]: Might be good to add something about how long is needed to get an event listed if we know? (Istr somebody mentioning this but I can't remember what they said.)

**Review by tdjames1 — APPROVED** [review]
Looks good, Neil, just suggesting minor punctuation change and comment about timeframe for university events listing.

File: .github/ISSUE_TEMPLATE/meeting_setup.md (hunk 2 above)
Comment (ns-rse) [reply .github/ISSUE_TEMPLATE/meeting_setup.md]: I recall reading this somewhere myself but can't find it now. 

I've put advice in the paragraph at the start of this section that it should be done two weeks in advance to maximimise notification for people to attend.

**Pull request merged**